test/browser.js

```diff
@@ -88,15 +88,16 @@ test('onDownloadProgress works', withPage, async (t, page) => {
 
 		const text = await window.ky(url, {
 			onDownloadProgress: (percent, transferredBytes, totalBytes, chunk) => {
-				data.push([percent, transferredBytes, totalBytes, chunk instanceof Uint8Array ? decodeUTF8(chunk) : chunk]);
+				const stringifiedChunk = chunk instanceof Uint8Array ? decodeUTF8(chunk) : String(chunk);
```

Comment:
We need to document in the readme and typings in what situations the `chunk` would be `undefined`.

Reply:
It's `undefined` only for the first call.